.github/workflows/e2e-tests.yml

```diff
@@ -2,9 +2,9 @@ name: e2e-tests
 
 on:
   push:
-    branches: [ main, release/*, feat/masthead-v2 ]
+    branches: [ main, release/*, feat/masthead-v2, feat/masthead-v2.1 ]
   pull_request:
-    branches: [ main, release/*, feat/masthead-v2 ]
+    branches: [ main, release/*, feat/masthead-v2, feat/masthead-v2.1 ]
```

Comment:
Addded so we get the CI check on this PR, not sure if we want to keep it. Not sure about the lifespan of this `feat/masthead-v2.1`, sounds like it's only matter of time before its merged down into `feat/masthead-v2-dev`. I imagine I'll also need to make adjustments here once the [merge conflict resolution happening over here](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/10031) makes it to `feat/masthead-v2.1`. We can adjust this once it's closer to ready.